src/Wortuhr.ino

```diff
@@ -1015,6 +1003,23 @@ void loop() {
         break;
     }
 
+    if (G.prog == COMMAND_MODE_WORD_CLOCK) {
+        calc_Word_array();
+
+        if (changes_in_array()) {
+            copy_array(Word_array, Word_array_old);
+            led_set(true);
+        } else if (Changes_in_Parameter) {
+            led_set();
+            Changes_in_Parameter = false;
```

Comment:
wenn sowohl `changes_in_array()` als auch `Changes_in_Parameter` wahr ist, dann muss `led_set()` trotzdem nur einmal aufgerufen werden, oder?
Dann kann das Zurücksetzen aus dem if-Block rausgenommen werden und einfach immer durchgeführt werden.

Reply:
Das ist richtig, man könnte das Zurücksetzten auch außerhalb der if-Anweisung immer durchführen. denn in beiden fällen werden die neunen Parameter angewandt.